### PR TITLE
[Feature #18] Add swagger to document REST API

### PR DESCRIPTION
**Pull request by Matthew-Kulich**

### CODE REVIEW

File: s-pipes-web/src/main/java/cz/cvut/spipes/config/SpringFoxConfig.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+                .paths(PathSelectors.any())
+                .build()
+                .apiInfo(apiInfo());
+    }
+
+    ApiInfo apiInfo() {
+        return new ApiInfoBuilder()
+                .title("SPipes Open API")
+                .description("SPipes Swagger Open API")
+                .version("v0.3.0")
```
Comment (blcham) [inline s-pipes-web/src/main/java/cz/cvut/spipes/config/SpringFoxConfig.java:31]: IMO, It is not a good idea to hardcode the version. 

If you want, you can set this up by configuration similarly as done here: https://github.com/opendata-mvcr/sgov/blob/3e27c6d0ad188cf086d3be1d620802c04b51938f/src/main/java/com/github/sgov/server/config/SpringFoxConfig.java

Another option is just removing this line and having no version there.

**Review by blcham — CHANGES_REQUESTED** [review]
LGTM, just issue with API version, see my comments

File: s-pipes-web/src/main/java/cz/cvut/spipes/config/SpringFoxConfig.java (hunk 1 above)
Comment (Matthew-Kulich) [reply s-pipes-web/src/main/java/cz/cvut/spipes/config/SpringFoxConfig.java]: For now I will delete that and later when we will migrate to spring boot I will add it.

**Review by blcham — APPROVED** [review]
LGTM

**Pull request merged**